quinn/src/connection.rs

```diff
@@ -963,7 +963,7 @@ pub(crate) struct State {
     endpoint_events: mpsc::UnboundedSender<(ConnectionHandle, EndpointEvent)>,
     pub(crate) blocked_writers: FxHashMap<StreamId, Waker>,
     pub(crate) blocked_readers: FxHashMap<StreamId, Waker>,
-    pub(crate) stopped: FxHashMap<StreamId, Waker>,
+    pub(crate) stopped: FxHashMap<StreamId, Arc<Notify>>,
```

Comment:
Nit: can you extract this and the related changes into a separate, earlier commit (that still compiles and passes tests at least locally)?

Reply:
Not sure I understand, what part of the overall commit should *not* be included in that commit then?

Reply:
I think the changes to the `stopped()` API can be in a separate commit from the internal API changes?

Reply:
Not sure how exactly that would work. If I change `State::stopped` to `Arc<Notify>`, I have to change `SendStream::poll_stopped`  as well, which is possible of course but not done in a minute because the `Notified` future has to be stored and pinned and polled in a loop. I can do but is this really worth it if the correct solution is already implemented?

Reply:
Okay, let's leave it in a single commit.